src/components/Users/UserForm.tsx

```diff
@@ -231,18 +238,13 @@ export default function UserForm({
     }),
     onSuccess: (resp: UserBase) => {
       toast.success(t("user_updated_successfully"));
-      queryClient.invalidateQueries({
-        queryKey: ["facilityUsers"],
-      });
-      queryClient.invalidateQueries({
-        queryKey: ["organizationUsers"],
-      });
-      queryClient.invalidateQueries({
-        queryKey: ["facilityOrganizationUsers"],
-      });
-      queryClient.invalidateQueries({
-        queryKey: ["getUserDetails", resp.username],
-      });
+      [
+        ["facilityUsers"],
+        ["organizationUsers"],
+        ["facilityOrganizationUsers"],
+        ["getUserDetails", resp.username],
+        ["currentUser"],
+      ].forEach((key) => queryClient.invalidateQueries({ queryKey: key }));
```

Comment:
AI bots won't like this...

the previous one was just cleaner/simpler and more readable

Reply:
AI bots will have to learn